CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ All notable changes to this project will be documented in this file. The format
 
 > **Note**: Changes and additions listed here are contained in the `:edge` image tag. These changes may not be as stable as released changes.
 
+### Added
+
+- New environment variable `MARK_SPAM_AS_READ` which if set to `1` marks incoming junk as read so no notification is generated
```

Comment:
```suggestion
- New environment variable `MARK_SPAM_AS_READ`. When set to `1`, marks incoming junk as "read" to avoid unwanted notification of junk as new mail ([#3489](https://github.com/docker-mailserver/docker-mailserver/pull/3489))
```